extensions.json

```
@@ -387,6 +394,12 @@
       "id": "neikeq.godot-csharp-vscode",
       "repository": "https://github.com/godotengine/godot-csharp-vscode"
     },
+    {
+      "id": "gharveymn.nightswitch-lite",
+      "repository": "https://github.com/gharveymn/nightswitch-lite",
+      "version": "2.2.2",
+      "checkout": "v2.2.2"
```

Same here, there is no `v2.2.2` tag in this repository.

Even weirder, the repository is still at version `2.2.1`: https://github.com/gharveymn/nightswitch-lite/blob/master/package.json even though the VSCode marketplace already has `2.2.2` -- where is the code for version `2.2.2`?

In any case, maybe let's not pin a specific version for this extension either, i.e. let's always auto-publish from the default branch as that's the best we can do:

```suggestion
      "repository": "https://github.com/gharveymn/nightswitch-lite"
```